packages/forma-36-react-components/src/components/Tabs/Tabs.css

```diff
@@ -30,6 +30,10 @@
   margin-right: 0;
 }
 
+.Tabs__with-divider {
+  border-bottom: 1px solid var(--color-element-mid);
```

Comment:
The following looks quite nice as an alternative, but it's a bigger change and maybe doesn't match anything else in the UI :D

```
box-shadow: 0 -3px 0 inset var(--color-element-mid);
```

![Screenshot 2019-08-21 at 16 07 40](https://user-images.githubusercontent.com/4446634/63439634-b4a89e80-c42e-11e9-9556-ffeb4df3e4a7.png)

Reply:
ahh that is what you meant that looks nice will do so!

Reply:
@m10l done